.python_package/setup.cfg

```diff
@@ -0,0 +1,20 @@
+[metadata]
+name = gridtools
```

Comment:
Should this be the same as the pypi package name? Because then it would conflict, and we should use something else e.g. 
```suggestion
name = gridtools-cpp
```